tests/Ssl/Generator/KeyPairGeneratorTest.php

```diff
@@ -28,12 +30,35 @@ public function setUp()
 
     public function test generateKeyPair generate random instance of KeyPair()
     {
-        $result = $this->service->generateKeyPair(1024);
+        $result = $this->service->generateKeyPair(new RsaKeyOption(1024));
```

Comment:
I think it would be great to keep a test using the deprecated way to ensure it still works.

Reply:
done